pkgs/development/tools/misc/nrf-command-line-tools/default.nix

```
@@ -0,0 +1,76 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+, udev
+, libusb1
+, segger-jlink
+}:
+
+let
+  supported = {
+    x86_64-linux = {
+      name = "linux-amd64";
+      sha256 = "1b69272c704f4526b8560fe3c64b92769626261c2a285258bde550d3dd218873";
+    };
+    aarch64-linux = {
+      name = "linux-arm64";
+      sha256 = "bcad32f9340c60374589e6613aa773dd9255e0cea7c50b9bd83ee47ca170a536";
+    };
+    armv7l-linux = {
+      name = "linux-armhf";
+      sha256 = "7b217b434e675dba4ae3a6f611fb9e15835bbe106fd1a4a4052ae68954111a2e";
```

```suggestion
      hash = "1b69272c704f4526b8560fe3c64b92769626261c2a285258bde550d3dd218873";
    };
    aarch64-linux = {
      name = "linux-arm64";
      hash = "bcad32f9340c60374589e6613aa773dd9255e0cea7c50b9bd83ee47ca170a536";
    };
    armv7l-linux = {
      name = "linux-armhf";
      hash = "7b217b434e675dba4ae3a6f611fb9e15835bbe106fd1a4a4052ae68954111a2e";
```
we should also use sri hashes here